### DISCUSSION TITLE
Can’t add Issue with the form so noting here - after PW2 update to 24.12.3

### DISCUSSION THREAD

**Discussion by Johndowne** [alandtse/tesla#968] [Q&A] [answered]
Integration on v3.21.0.
SOC reading shows 0% but batteries are full (100%) in reality.

**Comment by skipishere** [comment]
What's the issue you're having with the form? If you fill in all the required fields then it should work. 

Side note, also noticed my powerwall is also reporting 0 since 0100 GMT

**Reply by Johndowne** [reply]
It came back with not available at the moment try later type of words. Glad you’ve got the same issue (in the nicest possible way that is)

**Reply by skipishere** [reply]
Assuming this came back (did for me) so assuming it was a Telsa data issue

**Reply by Johndowne** [reply, marked as answer]
Yes indeed.
